<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="4" failed="2" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Testcase login started successfully]]>
    </line>
    <line>
      <![CDATA[Testcase loginTest started successfully]]>
    </line>
    <line>
      <![CDATA[Testcase loginTest started successfully]]>
    </line>
    <line>
      <![CDATA[Testcase loginTest started successfully]]>
    </line>
    <line>
      <![CDATA[Testcase loginTest started successfully]]>
    </line>
    <line>
      <![CDATA[Testcase addCustomer started successfully]]>
    </line>
  </reporter-output>
  <suite started-at="2023-07-14T00:05:56 IST" name="SDESeleniumFramework" finished-at="2023-07-14T00:07:34 IST" duration-ms="97866">
    <groups>
    </groups>
    <test started-at="2023-07-14T00:05:56 IST" name="LoginTest" finished-at="2023-07-14T00:07:34 IST" duration-ms="97866">
      <class name="com.SDESeleniumFramework.testCases.TC_LoginTest_001">
        <test-method is-config="true" signature="launchBrowser(java.lang.String)[pri:0, instance:com.SDESeleniumFramework.testCases.TC_LoginTest_001@73e9cf30]" started-at="2023-07-14T00:05:57 IST" name="launchBrowser" finished-at="2023-07-14T00:06:04 IST" duration-ms="7083" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method signature="login()[pri:0, instance:com.SDESeleniumFramework.testCases.TC_LoginTest_001@73e9cf30]" started-at="2023-07-14T00:06:04 IST" name="login" finished-at="2023-07-14T00:06:06 IST" duration-ms="1924" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Testcase login started successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:com.SDESeleniumFramework.testCases.TC_LoginTest_001@73e9cf30]" started-at="2023-07-14T00:06:06 IST" name="closeBrowser" finished-at="2023-07-14T00:06:06 IST" duration-ms="116" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- com.SDESeleniumFramework.testCases.TC_LoginTest_001 -->
      <class name="com.SDESeleniumFramework.testCases.TC_AddCustomerTest_003">
        <test-method is-config="true" signature="launchBrowser(java.lang.String)[pri:0, instance:com.SDESeleniumFramework.testCases.TC_AddCustomerTest_003@4f80542f]" started-at="2023-07-14T00:07:18 IST" name="launchBrowser" finished-at="2023-07-14T00:07:24 IST" duration-ms="6534" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method signature="addCustomer()[pri:0, instance:com.SDESeleniumFramework.testCases.TC_AddCustomerTest_003@4f80542f]" started-at="2023-07-14T00:07:24 IST" name="addCustomer" finished-at="2023-07-14T00:07:34 IST" duration-ms="9457" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Testcase addCustomer started successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- addCustomer -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:com.SDESeleniumFramework.testCases.TC_AddCustomerTest_003@4f80542f]" started-at="2023-07-14T00:07:34 IST" name="closeBrowser" finished-at="2023-07-14T00:07:34 IST" duration-ms="135" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- com.SDESeleniumFramework.testCases.TC_AddCustomerTest_003 -->
      <class name="com.SDESeleniumFramework.testCases.TC_LoginDDTTest_002">
        <test-method is-config="true" signature="launchBrowser(java.lang.String)[pri:0, instance:com.SDESeleniumFramework.testCases.TC_LoginDDTTest_002@534a5a98]" started-at="2023-07-14T00:06:06 IST" name="launchBrowser" finished-at="2023-07-14T00:06:32 IST" duration-ms="25791" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method signature="loginTest(java.lang.String,java.lang.String)[pri:0, instance:com.SDESeleniumFramework.testCases.TC_LoginDDTTest_002@534a5a98]" started-at="2023-07-14T00:06:33 IST" name="loginTest" data-provider="driveTest" finished-at="2023-07-14T00:06:55 IST" duration-ms="22652" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr512458]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[AbUnEbU]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Testcase loginTest started successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method signature="loginTest(java.lang.String,java.lang.String)[pri:0, instance:com.SDESeleniumFramework.testCases.TC_LoginDDTTest_002@534a5a98]" started-at="2023-07-14T00:06:55 IST" name="loginTest" data-provider="driveTest" finished-at="2023-07-14T00:07:01 IST" duration-ms="5485" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr512458]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[AbUnEbUw]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.SDESeleniumFramework.testCases.TC_LoginDDTTest_002.loginTest(TC_LoginDDTTest_002.java:31)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Testcase loginTest started successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method signature="loginTest(java.lang.String,java.lang.String)[pri:0, instance:com.SDESeleniumFramework.testCases.TC_LoginDDTTest_002@534a5a98]" started-at="2023-07-14T00:07:01 IST" name="loginTest" data-provider="driveTest" finished-at="2023-07-14T00:07:12 IST" duration-ms="10330" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr514684]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[yvematU]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Testcase loginTest started successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method signature="loginTest(java.lang.String,java.lang.String)[pri:0, instance:com.SDESeleniumFramework.testCases.TC_LoginDDTTest_002@534a5a98]" started-at="2023-07-14T00:07:12 IST" name="loginTest" data-provider="driveTest" finished-at="2023-07-14T00:07:17 IST" duration-ms="5478" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr514684]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[yvematUA]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.SDESeleniumFramework.testCases.TC_LoginDDTTest_002.loginTest(TC_LoginDDTTest_002.java:31)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Testcase loginTest started successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:com.SDESeleniumFramework.testCases.TC_LoginDDTTest_002@534a5a98]" started-at="2023-07-14T00:07:18 IST" name="closeBrowser" finished-at="2023-07-14T00:07:18 IST" duration-ms="186" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- com.SDESeleniumFramework.testCases.TC_LoginDDTTest_002 -->
    </test> <!-- LoginTest -->
  </suite> <!-- SDESeleniumFramework -->
</testng-results>
